sites/public/src/pages/sign-in.tsx

```diff
@@ -119,31 +121,84 @@ const SignIn = () => {
 
   useEffect(() => {
     if (
-      networkError?.error.response.data?.message ===
+      networkError?.error?.response?.data?.message ===
       EnumUserErrorExtraModelUserErrorMessages.accountNotConfirmed
     ) {
       setConfirmationStatusModal(true)
     }
   }, [networkError])
 
+  const MobileComponent = () => {
```

Comment:
There's some repetition here, could we (1) name this something more specific and (2) make it a function that we pass a flag to that changes just the class names?